CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ find_package (PkgConfig REQUIRED)
 SET (CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
 PKG_CHECK_MODULES(GLIB gio-unix-2.0>=2.46.0 REQUIRED)
 FIND_LIBRARY (RPMDB_LIBRARY NAMES rpmdb)
-find_package (LibSolv 0.6.21 REQUIRED COMPONENTS ext)
+find_package (LibSolv 0.6.30 REQUIRED COMPONENTS ext)
```

Comment:
This would break build for RHEL 7 which has libsolv 0.6.26

Reply:
But it shouldn't even be compiling against libsolv 0.6.26, because you broke it in b534bb224f4db35b1f96507d81cc65bb6ae7a5fb.

Reply:
That's right. Not objections to the patch then.